Dockerfile

```diff
@@ -1,54 +1,51 @@
+# build
```

Comment:
Can you provide the difference in build times and/or the image size when compared to the Alpine image?

Reply:
1.09GB for mealie-api:dev
886MB for python:3.9-buster
not sure what previous sizes were
maybe slim could be used but I've had some troubles with it: reinstalling build tools and dev libraries increase pretty much to the size of regular debian